syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.fandreuz.grpc.uds.benchmark.echo.helloworld";
option java_outer_classname = "HelloWorldProto";

package helloworld;

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  bytes payload = 1;
}

message HelloReply {
  bytes payload = 1;
}
